config/cesm/config_grids.xml

```diff
@@ -1306,6 +1312,13 @@
       <desc>user specified domain - only valid for DATM/CLM compset</desc>
     </domain>
 
+    <domain name="1x1_NEON">
+      <nx>1</nx> <ny>1</ny>
```

Comment:
The problem here that I ran into with CLM_USRDAT_NAME is that this domain file when updated will change ALL existing simulations being run with it. This is problematic. As such I came to the conclusion a better way to handle it is to have scripts that setup single-point create user-mod directories where the DOMAIN file is set to a particular path that includes a time-stamp in the filename. This way changes to files don't change the results of someone else's existing simulations.